admin-dev/themes/new-theme/js/pages/product/service/image.ts

```diff
@@ -73,8 +73,11 @@ export const saveImageInformations = async (
   data[`${formName}[apply_to_all_stores]`] = selectedFile.applyToAllStores;
 
   return $.ajax(saveUrl, {
-    method: 'PATCH',
-    data,
+    method: 'POST',
+    data: {
+      ...data,
+      _method: 'PATCH',
+    },
```

Comment:
Perfect, I believe there's also this method to adapt https://github.com/PrestaShop/PrestaShop/blob/c7879caed1803eca59cc2c2e2e634b4a94c04656/admin-dev/themes/new-theme/js/pages/product/service/image.ts#L116

Reply:
There had already been passages made it seems

https://github.com/PrestaShop/PrestaShop/blob/9faac2aeb7adffcb1d98ad5677a35bd775419040/admin-dev/themes/new-theme/js/pages/product/service/combination.ts#L75